var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -232,11 +235,15 @@ def setup_build_environment(self, env):
     # Note: cmake, build, and install stages are handled by CMakePackage
     ####################################################################
 
+    @property
+    def root_cmakelists_dir(self):
```

Comment:
This fixes warnings from cmake and is the mechanism spack provides to point to a non-root source directory. The previous way seems like it would be prone to errors.